### DISCUSSION TITLE
Hide posts that are indevelopment

### DISCUSSION THREAD

**Discussion by BenjamenSimon** [hugo-themes/toha#698] [Ideas]
Hello!

Is it possible to add a true/false toggle in the post front-matter to hide them from the website (sidebar, search, and recent posts)?

This would be useful while posts are in development so they can be hidden before release.

If I leave out the sidebar name they are hidden there but still show up in recent posts.

Thank you.

**Comment by hossainemruz** [comment]
Yes. Just add `draft: true` in the post front-matter. Ref: https://gohugo.io/variables/page/#page-variables

**Reply by BenjamenSimon** [reply]
Thank you :)
